src/main/res/values/strings.xml

```diff
@@ -96,12 +96,12 @@
     the applications it provides.</string>
   <string name="number_of_applications">Number of applications</string>
   <string name="ok">OK</string>
-  <string name="open_link_confirm_format">Open link %s?</string>
+  <string name="open_link_confirm_format">Open the %s link?</string>
```

Comment:
```suggestion
  <string name="open_link_confirm_format">Open %s?</string>
```
Making it clunky for everyone, for an error that could arise when links are long, is dubious. The better overall fix is to truncate the link somehow.